sdk/appconfiguration/Azure.Data.AppConfiguration/src/SettingSelector.cs

```diff
@@ -9,13 +9,11 @@ namespace Azure.Data.AppConfiguration
 {
     /// <summary>
     /// <see cref="SettingSelector"/> is a set of options that allows selecting
```

Comment:
I'm not sure, but you may need `<para>` tags on the first paragraph as well. The compiler likely doesn't care, but if it doesn't automagically add one because subsequent text has it (probably not), doc generates may not work correctly.